File: src/main/java/com/freya02/bot/versioning/supplier/BuildToolDependencies.java
package com.freya02.bot.versioning.supplier;

public interface BuildToolDependencies {
	String getBCDependencyFormatString();

	String getJDA4DependencyFormatString();

	String getJDA5DependencyFormatString();

	String getJDA5JitpackDependencyFormatString();
}

File: src/main/java/com/freya02/bot/versioning/supplier/BuildToolDependencies.kt
package com.freya02.bot.versioning.supplier

interface BuildToolDependencies {
    val bcDependencyFormatString: String
    val jda4DependencyFormatString: String
    val jda5DependencyFormatString: String
    val jdaJitpackDependencyFormatString: String
}